boilerplates/react-union-boilerplate-basic/src/widgets/Content/components/Root.js

```diff
@@ -4,8 +4,8 @@ import NestedComponent from './NestedComponent';
 
 const Root = ({ namespace, data }) => (
 	<div>
-		I am widget Content. With namespace: <b>{namespace}</b> and initial data:
-		<b>{JSON.stringify(data)}</b>
+		I am widget Content with namespace <b>{namespace}</b> and initial data
```

Comment:
with the

Reply:
I will rewrite this so it makes more sense.